bol/forum_service.php

```diff
@@ -1021,7 +1049,14 @@ public function findTopicPostCount( $topicId )
      */
     public function saveOrUpdatePost( $postDto )
     {
+        $oldPostInfo = $this->findPostById($postDto->id);

```

Comment:
Replace getting the old post with the `$dto->getEntinyUpdatedFields` method which returns updated field list